packages/interface/src/lib.ts

```diff
@@ -397,12 +397,27 @@ export interface PrincipalParser {
  * Library also provides utility functions for combining multiple
  * SignerImporters into one.
  */
-export interface SignerImporter<Self extends Signer = Signer> {
-  from(archive: SignerArchive<Self>): Self
+export interface SignerImporter<
+  ID extends DID = DID,
+  Code extends number = number
```

Comment:
It would be helpful to include a jsdoc `@template` comment for at least `Code`. I wouldn't necessarily know what kind of code you're talking about